VMD_scripts/blobulation.tcl

```diff
@@ -620,13 +731,68 @@ proc ::blobulator::blobUserAssign { blob1 MolID } {
 	
  }
 
-proc ::blobulator::blobUser2Assign { blob2 MolID } {
+proc ::blobulator::blobUserAssignSelector {blob1 MolID chainList} {
```

Comment:
This function appears to lack documentation

Reply:
Documentation has been added